	Use "#if HAVE_ALLOCA_H" instead of "#ifdef HAVE_ALLOCA_H"
	and similar, to avoid preprocessor warnings.

diff -ur dvipng-1.12.orig/color.c dvipng-1.12/color.c
--- dvipng-1.12.orig/color.c	2008-06-03 19:29:31.000000000 +0200
+++ dvipng-1.12/color.c	2009-03-25 14:06:21.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 
diff -ur dvipng-1.12.orig/dvipng.h dvipng-1.12/dvipng.h
--- dvipng-1.12.orig/dvipng.h	2008-06-03 19:30:18.000000000 +0200
+++ dvipng-1.12/dvipng.h	2009-03-25 14:04:41.000000000 +0100
@@ -478,11 +478,11 @@
 
 /************************timing stuff*********************/
 #ifdef TIMING
-#if TIME_WITH_SYS_TIME
+#ifdef TIME_WITH_SYS_TIME
 # include <sys/time.h>
 # include <time.h>
 #else
-# if HAVE_SYS_TIME_H
+# ifdef HAVE_SYS_TIME_H
 #  include <sys/time.h>
 # else
 #  include <time.h>
diff -ur dvipng-1.12.orig/fontmap.c dvipng-1.12/fontmap.c
--- dvipng-1.12.orig/fontmap.c	2008-05-05 20:58:00.000000000 +0200
+++ dvipng-1.12/fontmap.c	2009-03-25 14:04:41.000000000 +0100
@@ -25,7 +25,7 @@
 #include "dvipng.h"
 
 static struct filemmap psfont_mmap;
-#if HAVE_FT2
+#ifdef HAVE_FT2
 static struct filemmap ttfont_mmap;
 #endif
 static struct psfontmap *psfontmap=NULL;
@@ -106,10 +106,10 @@
     newentry->psfile = copyword(copyfrom->psfile);
     newentry->encname = copyword(copyfrom->encname);
     newentry->encoding = copyfrom->encoding;
-#if HAVE_LIBT1
+#ifdef HAVE_LIBT1
     newentry->t1_transformp = copyfrom->t1_transformp;
 #endif
-#if HAVE_FT2
+#ifdef HAVE_FT2
     newentry->ft_transformp = copyfrom->ft_transformp;
     newentry->subfont = copyfrom->subfont;
 #endif
@@ -120,10 +120,10 @@
     newentry->psfile = NULL;
     newentry->encname = NULL;
     newentry->encoding = NULL;
-#if HAVE_LIBT1
+#ifdef HAVE_LIBT1
     newentry->t1_transformp = NULL;
 #endif
-#if HAVE_FT2
+#ifdef HAVE_FT2
     newentry->ft_transformp = NULL;
     newentry->subfont = NULL;
 #endif
@@ -249,14 +249,14 @@
 	}
 	free(word);
       }
-#if HAVE_FT2
+#ifdef HAVE_FT2
       entry->ft_transformp=&(entry->ft_transform);
       entry->ft_transform.xx=(FT_Fixed)(cxx*0x10000);
       entry->ft_transform.xy=(FT_Fixed)(cxy*0x10000);
       entry->ft_transform.yx=0;
       entry->ft_transform.yy=0x10000;
 #endif
-#if HAVE_LIBT1
+#ifdef HAVE_LIBT1
       entry->t1_transformp=&(entry->t1_transform);
       entry->t1_transform.cxx=cxx;
       entry->t1_transform.cxy=cxy;
diff -ur dvipng-1.12.orig/ft.c dvipng-1.12/ft.c
--- dvipng-1.12.orig/ft.c	2008-02-08 14:45:24.000000000 +0100
+++ dvipng-1.12/ft.c	2009-03-25 14:04:41.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 
diff -ur dvipng-1.12.orig/misc.c dvipng-1.12/misc.c
--- dvipng-1.12.orig/misc.c	2008-06-02 21:29:33.000000000 +0200
+++ dvipng-1.12/misc.c	2009-03-25 14:05:54.000000000 +0100
@@ -195,7 +195,7 @@
 	}
 	break ;
       case 't':       /* specify paper format, only for cropmarks */
-#if HAVE_GDIMAGECREATETRUECOLOR
+#ifdef HAVE_GDIMAGECREATETRUECOLOR
 	/* Truecolor */
 	if (strncmp(p,"ruecolor",8)==0) { 
 	  if (p[8] != '0') {
diff -ur dvipng-1.12.orig/pk.c dvipng-1.12/pk.c
--- dvipng-1.12.orig/pk.c	2008-05-05 20:58:00.000000000 +0200
+++ dvipng-1.12/pk.c	2009-03-25 14:05:54.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 
diff -ur dvipng-1.12.orig/set.c dvipng-1.12/set.c
--- dvipng-1.12.orig/set.c	2008-02-08 14:45:23.000000000 +0100
+++ dvipng-1.12/set.c	2009-03-25 14:06:04.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 #include <math.h>
diff -ur dvipng-1.12.orig/t1.c dvipng-1.12/t1.c
--- dvipng-1.12.orig/t1.c	2008-02-08 14:45:23.000000000 +0100
+++ dvipng-1.12/t1.c	2009-03-25 14:06:12.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 
diff -ur dvipng-1.12.orig/tfm.c dvipng-1.12/tfm.c
--- dvipng-1.12.orig/tfm.c	2008-05-05 20:58:00.000000000 +0200
+++ dvipng-1.12/tfm.c	2009-03-25 14:06:21.000000000 +0100
@@ -23,7 +23,7 @@
 ************************************************************************/
 
 #include "dvipng.h"
-#if HAVE_ALLOCA_H
+#ifdef HAVE_ALLOCA_H
 # include <alloca.h>
 #endif
 
